lib/fathead/mdn_css/parse.pl

```diff
@@ -215,27 +198,44 @@ sub clean_string {
     return $clean;
 }
 
+sub build_initial_value {
+    my ($content) = @_;
+    my $initial_value = "<p><b>Initial Value: </b><em>$content</em></p>";
+    return $initial_value;
+}
+
 sub build_abstract {
-    my ( $description, $code ) = @_;
+    my ( $description, $code, $initial_value ) = @_;
     say "NO DESCRIPTION!" if $description eq "";
+    $description = trim($description);
+    $description =~ s/\r?\n+/\\n/g;
+    $initial_value =~ s/\r?\n+/\\n/g if $initial_value;
+    $code = clean_code($code) if $code;
     my $out;
-    $out .= "<p>$description</p>" if $description;
-    $out .= "<br><pre><code>$code</code></pre>" if $code;
+    $out .= "<p>$description</p>"           if $description;
+    $out .= "$initial_value"                if $initial_value;
+    $out .= "<pre><code>$code</code></pre>" if $code;
     return $out;
 }
 
-sub make_article {
+sub make_and_write_article {
     my ( $title, $description, $link, $external_links ) = @_;
     say '';
     say "TITLE: $title";
     say "LINK: $link";
-    say "DESCRIPTION: $description" if $description;
+    say "DESCRIPTION: $description"       if $description;
     say "EXTERNAL LINKS $external_links " if $external_links;
-    my @data = (
-        $title, 'A', '', '', '', '', '', '', $external_links || '',
+    my $title_clean = clean_string($title);
+    my @data        = join "\t",
+      (
+        $title_clean, 'A', '', '', '', '', '', '', $external_links || '',
```

Comment:
I think we should actually make the Abstract with `title` and create a redirect for `title_clean`. The benefit being the final article title that's displayed will contain the non-alphanumeric character that are expected.

e.g.

`import` redirects to `@import` so `css import` shows a detail pane where the `title` is `@import` instead of `import`

likewise `spelling-error` redirects to `::spelling-error`